jbang/azure-eventhubs-kafka-azure-schema-registry/README.md

```diff
@@ -0,0 +1,44 @@
+# Example for consuming from EventHubs in Avro format, using Azure Schema Registry
```

Comment:
Should we say Azure EventHubs ?

Reply:
We could fix it later. I think we could merge in this form and then makes some changes.

Reply:
ok